src/base/bittorrent/session.cpp

```diff
@@ -2368,6 +2368,36 @@ void Session::setIPFilterFile(QString path)
     }
 }
 
+void Session::setBannedIPs(const QStringList &newList)
+{
+    if (newList == m_bannedIPs)
```

Comment:
"Starting with Qt 5.0, you are instead encouraged to use directly the implementations available in the STL"
http://doc.qt.io/qt-5/qtalgorithms.html#qt-and-the-stl-algorithms